package.json

```diff
@@ -21,7 +21,7 @@
   "main": "./metasync.js",
   "readmeFilename": "README.md",
   "scripts": {
-    "test": "node tests/examples && node tests/chain && npm run lint",
+    "test": "jest && npm run lint",
```

Comment:
Yeah, and one more thing. It doesn't seem like a good idea to remove the old tests until all of them are converted. Keep them and make this `jest && node tests/examples && ...`.